asab/web/authn/oauth.py

```diff
@@ -4,25 +4,31 @@
 import asab.web.rest
 
 
-async def oauthclient_middleware_factory(request, handler, url):
-	authorization = request.headers.get("Authorization", None)
-	if authorization is None:
+def oauthclient_middleware_factory(app, *args, url, **kwargs):
```

Comment:
`url` is probably too generic name ... add a comment or change a name to something more obvious (aka URL of what?)